charts/spire/charts/spire-server/templates/statefulset.yaml

```diff
@@ -160,19 +160,21 @@ spec:
             {{- toYaml .Values.controllerManager.securityContext | nindent 12 }}
           image: {{ template "spire-lib.image" (dict "appVersion" $.Chart.AppVersion "image" .Values.tornjak.image "global" .Values.global) }}
           imagePullPolicy: {{ .Values.tornjak.image.pullPolicy }}
+          {{- if eq (.Values.tornjak.config.connectionType | toString) "http" }}
           startupProbe:
             httpGet:
               scheme: HTTP
               path: /api/tornjak/serverinfo
               port: 10000
             {{- toYaml .Values.tornjak.startupProbe | nindent 12 }}
+          {{- end }}
           args:
             - --spire-config
             - /run/spire/config/server.conf
             - --tornjak-config
             - /run/spire/tornjak-config/server.conf
           ports:
-            - name: tornjak
+            - name: {{ include "spire-tornjak.portname" . }}
```

Comment:
Do we need to change the port name? It will complicate the ingresses as well. 

Would be better if we can just point the ingresses at the tornjak service on the http port.

More logical would be 2 ports on the service:

- http
- https

Tornjak itself should listen on both and do a perm redirect as soon you enable TLS.

That way you don't break existing web behavior, where end users are automatically redirected to the https port as soon it is enabled in tornjak.